packages/sanes-chrome-extension/src/routes/tx-request/index.dom.spec.ts

```diff
@@ -12,44 +14,41 @@ import {
   clickOnRejectButton,
   confirmRejectButton,
 } from './test/operateTXRequest';
-import { travelToTXRequest } from './test/travelToTXRequest';
 
 describe('DOM > Feature > Transaction Request', (): void => {
+  const requests: ReadonlyArray<Request> = [
+    {
+      id: 1,
+      type: 'signAndPost',
+      reason: 'Test get Identities',
+      data: {
+        senderUrl: 'http://finnex.com',
+        creator: '0x873fAA4cdDd5b157e8E5a57e7a5479AFC5aaaaaa' as Address,
+        tx: getTransaction(),
+      },
+      accept: jest.fn(),
+      reject: jest.fn(),
+    },
+  ];
+
   let store: Store<RootState>;
   let identityDOM: React.Component;
 
   beforeEach(async () => {
     store = aNewStore();
-    const requests: ReadonlyArray<Request> = [
-      {
-        id: 1,
-        type: 'signAndPost',
-        reason: 'Test get Identities',
-        data: {
-          senderUrl: 'http://finnex.com',
-          creator: '0x873fAA4cdDd5b157e8E5a57e7a5479AFC5aaaaaa' as Address,
-          tx: getTransaction(),
-        },
-        accept: jest.fn(),
-        reject: jest.fn(),
-      },
-    ];
     identityDOM = await travelToTXRequest(store, requests);
-  });
+  }, 60000);
 
   it('should accept incoming request and redirect to account status view', async (): Promise<void> => {
     const inputs = TestUtils.scryRenderedDOMComponentsWithTag(identityDOM, 'button');
 
     expect(inputs.length).toBe(2);
 
     const acceptButton = inputs[0];
-
-    TestUtils.act(() => {
```

Comment:
Can you confirm me those tests do not generate any warning in console?

Reply:
Yes, they do not generate any warning